docs/api/alert.md

```diff
@@ -160,6 +160,16 @@ import Customization from '@site/static/usage/alert/customization/index.md';
  If you are building an Ionic Angular app, the styles need to be added to a global stylesheet file.
 :::
 
+## Accessibility
+
+Ionic automatically sets the Alert's `role` to either `alertdialog` or `alert`, depending on whether any buttons or inputs are included.
```

Comment:
Can we update the wording to make it clear that having any buttons/inputs will cause the role to be `alertdialog`?